website/versioned_docs/version-2.0.0-beta.14/deployment.mdx

```diff
@@ -394,8 +409,12 @@ jobs:
         with:
           node-version: 14.x
           cache: yarn
+
+      - name: Get yarn cache
+        id: yarn-cache
+        run: echo "::set-output name=dir::$(yarn cache dir)"
```

Comment:
Why? I don't understand what's the significance of this step...

Reply:
Just tested again, and this lines were not necessary indeed, they were added while debugging and erroneously maintained. Will rm.